apps-rendering/src/components/Metadata/Metadata.defaults.tsx

```diff
@@ -0,0 +1,42 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { remSpace } from '@guardian/source-foundations';
+import Avatar from 'components/Avatar';
+import Byline from 'components/Byline';
+import CommentCount from 'components/CommentCount';
+import Dateline from 'components/Dateline';
+import Follow from 'components/Follow';
+import { getFormat } from 'item';
+import type { Item } from 'item';
+import type { FC } from 'react';
+
+export const defaultStyles = css`
+	display: flex;
+	margin-bottom: ${remSpace[5]};
+`;
+
+export const defaultTextStyles = (withByline: boolean): SerializedStyles => css`
+	flex-grow: 1;
+	padding-top: ${withByline ? remSpace[1] : 0};
+`;
+
+type Props = {
+	item: Item;
+	withByline: boolean;
+};
+
+export const DefaultMetadata: FC<Props> = ({ item, withByline }: Props) => (
+	<div css={defaultStyles}>
```

Comment:
I think the pattern you've used in other places is to have styles come in as a `className` prop, and then consumers pass in the `defaultStyles` to that prop? The `Styleable` type can be used to automate creating the Emotion props for this:

https://github.com/guardian/dotcom-rendering/blob/94b90145584680365ebd327570205c174ce9d846/apps-rendering/src/lib.ts#L19-L22

Here's an example of it in use:

https://github.com/guardian/dotcom-rendering/blob/94b90145584680365ebd327570205c174ce9d846/apps-rendering/src/components/HeadingTwo/HeadingTwo.defaults.tsx#L17-L23

Reply:
Yep, totally true, can do that for this. The annoying thing is that there's two sets of styles, on the main component and on the text. So I can pass the `css` prop to the component and it'll be magically transmogrified to `className` but the text styles will stay of type `SerializedStyles`. Which is annoying, and the reason I was leaving this until after the discussion I raised in the description.